server/src/api.js

```diff
@@ -59,8 +61,15 @@ export function buildFastify(renderFn, compareFn) {
       const viewport = request.body.viewport;
       const fullpage = request.body.fullpage ?? false;
       const style = request.body.style;
+      const usehMhtmlConverter = request.body.usehMhtmlConverter ?? false;
```

Comment:
I appreciate the backwards compatible approach, but would use the MHTML->HTML path by default because MHTML simply produces the wrong results. If things work properly, I would even suggest to get rid of the `usehMhtmlConverter` option entirely to keep things simple

Reply:
I changed the default on `main`